src/kvstore/kvstore_dist.h

```diff
@@ -131,17 +139,51 @@ class KVStoreDist : public KVStoreLocal {
   }
 
  private:
+  /**
+   * \brief struct for ps keys and lens
+   */
+  struct PSKV {
```

Comment:
Use more descriptive names

Reply:
Hmm, this is an existing struct, I moved it up so functions below can understand the variable. I kept it the same